bcda/auth/middleware.go

```diff
@@ -71,7 +71,13 @@ func ParseToken(next http.Handler) http.Handler {
 			// okta token
 			switch claims.Issuer {
 			case "ssas":
-				ad, _ = adFromClaims(repository, claims)
+				ad, err = adFromClaims(repository, claims)
+				if err != nil {
+					log.Error(err)
+					next.ServeHTTP(w, r)
```

Comment:
Can we update this to return the 401 status code here instead of moving on to the next handler? Like we are doing [here](https://github.com/CMSgov/bcda-app/blob/master/bcda/auth/middleware.go#L120).

I think this'll be a good opportunity for us to update the other error scenarios in ParseToken

Reply:
Great idea!